src/pages/Popup/components/CustomForm.jsx

```diff
@@ -95,6 +97,21 @@ class AddAlarmForm extends Component {
     this.check();
   };
 
+  handleEndTimeChange = async (event) => {
+    var val = event.target.value;
+    var d = new Date(val);
+    var error = false;
+    console.log(val, d);
+    if (val === '' || d < new Date()) {
```

Comment:
Add a check if the end time is greater than the start time or not. Currently, I can enter an end time which is before the start time. Have a look:
![autoleave](https://user-images.githubusercontent.com/56914284/122766769-83fe4e00-d2bf-11eb-92d5-2168c58c3770.png)